src/room/GroupCallView.tsx

```diff
@@ -97,7 +97,7 @@ export const GroupCallView: FC<Props> = ({
   const { displayName, avatarUrl } = useProfile(client);
   const roomName = useRoomName(rtcSession.room);
   const roomAvatar = useRoomAvatar(rtcSession.room);
-  const { perParticipantE2EE, returnToLobby } = useUrlParams();
```

Comment:
Does this mean we no longer respect the perParticipantE2EE URL parameter? Can it be removed?

Reply:
This isn't the intention. I will have a look and check that this isn't the case.

Reply:
I've removed this part of the PR for now.